mne/io/quspin/tests/data/quspin_N1_test_data.lvm

```diff
@@ -0,0 +1,62 @@
+LabVIEW Measurement	
```

Comment:
We have moved away from storing test data in MNE-Python, could you add the file to https://github.com/mne-tools/mne-testing-data ?